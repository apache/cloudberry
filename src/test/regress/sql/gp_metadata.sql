select version() ~ '^PostgreSQL (1[0-9]+)(\.[0-9]+)?(devel|stable)?(beta[0-9])? \(Apache Cloudberry ([0-9]+\.){2}[0-9]+.+' as version;
select gp_opt_version() ~ '^(GPOPT version: 4.0.0, Xerces version: ([0-9]+\.){2}[0-9]+|Server has been compiled without ORCA)$' as version;
