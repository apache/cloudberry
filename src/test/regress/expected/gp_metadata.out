select version() ~ '^PostgreSQL (1[0-9]+)(\.[0-9]+)?(devel|stable)?(beta[0-9])? \(Apache Cloudberry ([0-9]+\.){2}[0-9]+.+' as version;
 version 
---------
 t
(1 row)

select gp_opt_version() ~ '^(GPOPT version: 4.0.0, Xerces version: ([0-9]+\.){2}[0-9]+|Server has been compiled without ORCA)$' as version;
 version 
---------
 t
(1 row)

